example/hello.cc

```diff
@@ -1,6 +1,6 @@
 module;
 
-#include <cstdio>
+import <cstdio>;
```

Comment:
Let's keep this and introduce another example/test that uses standard imports. The new example can be only tested on supported compiler(s).